samples/logging/custom-factory/sample.md

```diff
@@ -49,4 +49,13 @@ The implementation of `ILog` handles writing the entries and the log filtering.
 
 ### Enabling Logging
 
-snippet:ConfigureLogging
\ No newline at end of file
+snippet:ConfigureLogging
+
+
+## Verifying that the sample works correctly
+
+In this sample the information at the Info level is logged to the console window.
+
+There should be a few standard logging entries in the console window that are automatically created by NServiceBus when logging level is set to Info, for example `Queue [private$\error] is running with [Everyone] and [NT AUTHORITY\ANONYMOUS LOGON] permissions. Consider setting appropriate permissions, if required by the organization. For more information, consult the documentation.`
```

Comment:
> should this be an include?

It could, but then if you want to change it you need to review all places where it's used, might become inaccurate. So that's why I went with copying, but no strong feelings about it.

> i prefer the word "will" over "should"

Ok, i'll adjust